src/ray/gcs/pubsub/test/gcs_pub_sub_test.cc

```diff
@@ -106,6 +109,7 @@ class GcsPubSubTest : public ::testing::Test {
   std::shared_ptr<gcs::RedisClient> client_;
   const std::chrono::milliseconds timeout_ms_{60000};
   std::shared_ptr<gcs::GcsPubSub> pub_sub_;
+  absl::Mutex vector_mutex_;
```

Comment:
Can you also add `GUARDED_BY` to the vars protected by this mutex?

Reply:
no problem